src/transformers/models/bridgetower/modeling_bridgetower.py

```diff
@@ -1868,7 +1868,7 @@ def forward(
 
         logits = torch.stack([text_embeds, image_embeds, cross_embeds], dim=-2)
 
-        logit_scale = self.logit_scale.exp()
+        logit_scale = self.logit_scale.exp().to(device=text_embeds.device)
```

Comment:
Here I assume `text_embeds`, `image_embeds`, `cross_embeds` are all on the same device, and the test pass several times. In general, I am not sure  if this always holds.

Reply:
You should make sure they are by casting them all to the same device.